pyk/src/pyk/__main__.py

```diff
@@ -315,6 +315,7 @@ def exec_kompile(options: KompileCommandOptions) -> None:
         'gen_glr_bison_parser': options.gen_glr_bison_parser,
         'bison_lists': options.bison_lists,
         'outer_parsed_json': options.outer_parsed_json,
+        'ignore_warnings': options.ignore_warnings,
```

Comment:
To fix the test failures, now that the option has been moved to `__call__`, this value needs to be passed to the call at line 344 instead:

```python3
try:
    Kompile.from_dict(kompile_dict)(
        output_dir=kompiled_directory,
        ...
```